src/components/MainView.js

```diff
@@ -33,19 +33,20 @@ export default class MainView extends React.Component {
   }
 
   export(json) {
-    let csv  = Papa.unparse(json),
-        blob = new Blob([csv], {type: 'text/csv'}),
-        url  = window.URL.createObjectURL(blob),
-        link = document.createElement('a'),
-        evt  = document.createEvent('MouseEvents');
-
-    link.setAttribute('href', url);
-    link.setAttribute('target', '_blank');
-    link.setAttribute('download', 'data.csv');
-
-    evt.initMouseEvent('click', true, true, document.defaultView, 1, 0, 0, 0, 0,
-      false, false, false, false, 0, null);
-    link.dispatchEvent(evt);
+    var filename = "data.csv";
```

Comment:
In ES6 we have `let` and `const`, which provide much more obvious block level scoping. Let's also prefer `const` where we can: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.srb3gi4wz